src/azure-cli/azure/cli/command_modules/eventgrid/custom.py

```diff
@@ -595,7 +595,7 @@ def cli_system_topic_create_or_update(
         tags=None,
         identity=None):
 
-    identity_info = _get_identity_info(identity)
+    identity_info = _get_identity_info_only_if_not_none(identity)
```

Comment:
Does domain_create_or_update need the same change(L253)?
https://github.com/Azure/azure-cli/blob/8c04af8c0bd508b00c28a21178a6e713060fbbb6/src/azure-cli/azure/cli/command_modules/eventgrid/custom.py#L231-L262

Reply:
no. this change is needed under some system-topics which are global.